src/main/java/internet/shop/dao/impl/CarDaoImpl.java

```diff
@@ -1,54 +1,287 @@
+            preparedStatementOfCar.executeUpdate();
+
+            ResultSet generatedKeys = preparedStatementOfCar.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't create the data - "
+                    + car + " into cars", exception);
+        }
         return car;
     }
 
     @Override
     public Optional<Car> get(Long id) {
-        return Storage.getCars().stream()
-                .filter(x -> x.getId().equals(id))
-                .findFirst();
+        Optional<Car> optionalCar;
+        String queryCreateCarAndManufacturer = "select cars.id "
+                + "as c_id, cars.manufacturer_id as c_mid, "
+                + "cars.model as c_m, manufacturers.id as m_id, manufacturers.name as m_n, "
+                + "manufacturers.country as m_c from cars "
+                + "inner join manufacturers on manufacturers.id = cars.manufacturer_id "
+                + "where cars.deleted = false and manufacturers.deleted = false and cars.id = ?;";
+        String queryCreateDriver = "select cars.id as c_id, drivers.id as d_id, "
+                + "drivers.name as d_n, drivers.license_number as d_ln from cars "
+                + "inner join cars_drivers on cars_drivers.car_id = cars.id "
+                + "inner join drivers on drivers.id = cars_drivers.driver_id "
+                + "where cars.id = ? and drivers.deleted = false;";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement stmtCarManufacturer = connection
+                        .prepareStatement(queryCreateCarAndManufacturer);
+                PreparedStatement stmtDriver = connection
+                        .prepareStatement(queryCreateDriver)) {
+            stmtCarManufacturer.setLong(1, id);
+            stmtDriver.setLong(1, id);
+
+            ResultSet resultSetCar = stmtCarManufacturer.executeQuery();
+            ResultSet resultSetDriver = stmtDriver.executeQuery();
+            Car car = new Car();
+            if (resultSetCar.next()) {
+                car = createCar(resultSetCar);
+                Manufacturer manufacturer = createManufacturer(resultSetCar);
+                car.setManufacturer(manufacturer);
+            }
+            while (resultSetDriver.next()) {
+                Driver driver = creteDriver(resultSetDriver);
+                car.setDrivers(driver);
+            }
+            optionalCar = Optional.of(car);
+        } catch (SQLException exception) {
+            throw new DataProcessingException("Can't get the data by id: " + id, exception);
+        }
+        return optionalCar;
     }
 
     @Override
     public List<Car> getAll() {
-        return Storage.getCars();
+        List<Car> carList = new ArrayList<>();
+        String query = "SELECT id from cars where deleted = false";
```

Comment:
please use the same approach with 2 queries and `getDriversByCarId()` in this method

Reply:
done